jcl/src/jdk.attach/share/classes/com/ibm/tools/attach/attacher/OpenJ9AttachProvider.java

```diff
@@ -88,6 +91,11 @@ public OpenJ9VirtualMachine attachVirtualMachine (
 	@Override
 	public List<VirtualMachineDescriptor> listVirtualMachines() {
 
+		PrivilegedAction<List<VirtualMachineDescriptor>> action = () -> listVirtualMachinesImp();
+		return AccessController.doPrivileged(action);
```

Comment:
This should re-throw RuntimeException and Error similarly to attachTarget().

Reply:
Fixed.